_data/simple-icons.json

```diff
@@ -8875,7 +8875,7 @@
         {
             "title": "Lerna",
             "hex": "C084FC",
-            "source": "https://user-images.githubusercontent.com/900523/173044458-fd0b57f6-6374-4265-98b5-eb8f55fe1fb3.svg"
+            "source": "https://github.com/lerna/website/blob/98d8454238319ad3203ee6947d983d7a0b3703d9/images/lerna-logo-dark.svg"
```

Comment:
This is not correct. There is no `L` in the source.

See https://github.com/simple-icons/simple-icons/pull/12092#discussion_r1835834125.